src/main/java/model/generator/ModelGenerator.java

```diff
@@ -0,0 +1,92 @@
+package model.generator;
+
+import model.structures.Model;
+
+import java.util.Random;
+
+public class ModelGenerator {
```

Comment:
Czy ta klasa jest potrzebna? Jak nie to ja wywal

Reply:
jest potrzebna